### PR TITLE
Change php7 errors to exception to maintain compatability with function declaration #264

### PR DESCRIPTION
**Pull request by jamiehd**

### CODE REVIEW

File: src/DebugBar/DataCollector/ExceptionsCollector.php [hunk 1]
```diff
@@ -29,6 +31,9 @@ public function addException(Exception $e)
     {
         $this->exceptions[] = $e;
         if ($this->chainExceptions && $previous = $e->getPrevious()) {
+            if($previous instanceof Error) {
```
Comment (GrahamCampbell) [inline src/DebugBar/DataCollector/ExceptionsCollector.php]: Better to say not instance of exception

File: src/DebugBar/DataCollector/ExceptionsCollector.php [hunk 2]
```diff
@@ -88,17 +93,11 @@ public function formatExceptionData(Exception $e)
         );
     }
 
-    /**
-     * @return string
```
Comment (GrahamCampbell) [inline src/DebugBar/DataCollector/ExceptionsCollector.php]: ?

File: src/DebugBar/DataCollector/ExceptionsCollector.php [hunk 3]
```diff
@@ -88,17 +93,11 @@ public function formatExceptionData(Exception $e)
...
-     */
     public function getName()
     {
         return 'exceptions';
     }
 
-    /**
-     * @return array
```
Comment (GrahamCampbell) [inline src/DebugBar/DataCollector/ExceptionsCollector.php]: ?

File: src/DebugBar/DataCollector/ExceptionsCollector.php [hunk 4]
```diff
@@ -10,7 +10,9 @@
 
 namespace DebugBar\DataCollector;
 
+use Error;
 use Exception;
+use Symfony\Component\Debug\Exception\FatalThrowableError;
```
Comment (reyostallenberg) [inline src/DebugBar/DataCollector/ExceptionsCollector.php]: Where is the requirement for this?

File: src/DebugBar/DataCollector/ExceptionsCollector.php (hunk 3 above)
Comment (jamiehd) [inline src/DebugBar/DataCollector/ExceptionsCollector.php]: Welp, didn't even see that. I'll get it sorted now. I blame my crazy-efficient IDE.

File: src/DebugBar/DataCollector/ExceptionsCollector.php (hunk 4 above)
Comment (jamiehd) [inline src/DebugBar/DataCollector/ExceptionsCollector.php]: Good spot. Thanks

File: src/DebugBar/DataCollector/ExceptionsCollector.php (hunk 1 above)
Comment (barryvdh) [inline src/DebugBar/DataCollector/ExceptionsCollector.php]: Why? Should have same effect right?

**Pull request closed without merging**